dashboard/src/components/PreviousQueryList.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React, { Component } from "react";
+
+import PreviousQuery from "./PreviousQuery";
+
+import "../assets/css/App.css";
+
+class PreviousQueryList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filterText: "",
+    };
+  }
+
+  filterQueries = function() {
+    var that = this;
+    return this.props.queries.filter(function(item) {
```

Comment:
if we use fat arrow function in the callback, can we stop using `var that = this`?